src/commands/m1.ts

```diff
@@ -0,0 +1,19 @@
+import { Command } from '../types';
+
+const nameA = 'm1';
+
+const m1: Command = {
+  name: `${nameA}`,
```

Comment:
Dlaczego tak? Nie lepiej dać po prostu name na `"m1"`?

Reply:
Ups. Babrałem się w tym, bo mi cyferek nie przyjmowało. Dopiero po czasie zobaczyłem, że to w regexie trzeba pomajstrować i zapomniałem tego usunąć. Już poprawione.